benchmarks/benchmark_autoround.py

```diff
@@ -0,0 +1,102 @@
+import argparse
+import logging
+
+import torch
+import torchao.prototype.autoround.utils as ar_utils
+from torchao.prototype.autoround.autoround_demo import quantize_model_with_autoround
+from torchao.prototype.autoround.core import auto_round_config
+from torchao.utils import benchmark_model, TORCH_VERSION_AFTER_2_4
```

Comment:
nit: `TORCH_VERSION_AFTER_2_4` is renamed to `TORCH_VERSION_AT_LEAST_2_4` now, please rebase on main branch

Reply:
Thanks, updated.